staff/carlos-corredor/cv/carlos-c-cv.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV Carlos C</title>
+</head>
+<body>
+    <header>
+      <h2>Carlos Wilfredo Corredor García</h2>
+      <h3><em>Ingeniero Geofísico</em></h3>
+      <h3><em>Técnico Medio en Electrónica</em></h3>
+      <br>
+      <p><strong>Dirección de habitación:</strong> Gran vía 586, 1º 2º, Barcelona, España</p>
+      <p><strong>Cel:</strong> +58 416 - 4022957</p>
+      <p><strong>Cel:</strong> +34 635 - 558867</p>
+    </header>
+    <br>
+    <section>
+        <p><strong>CI:</strong> V-14.381.184</p>
+        <p><strong>...</strong> </p>
+    </section>
+    <main>
+        <section>
+            <h4><em>Estudios Realizados</em></h4>
+            <ul>
+                <li><ul>Pregrado: Universidad Central de Venezuela (UCV), Facultad de Ingeniería, Departamento de Geofísica, 2013.
```

Comment:
@Carlos7979 I think this <ul> it's useless and it has not closing tag.

Reply:
The goal is to create a list within the list so that the bullets are different indicating a different level.
Now I closed the tags. Thank you.